src/UserSpaceInstrumentation/TrampolineTest.cpp

```diff
@@ -623,7 +623,7 @@ class InstrumentFunctionTest : public testing::Test {
     ORBIT_CHECK(!return_trampoline_or_error.value()->EnsureMemoryExecutable().has_error());
 
     // Copy the beginning of the function over into this process.
-    constexpr uint64_t kMaxFunctionPrologueBackupSize = 20;
+    constexpr uint64_t kMaxFunctionPrologueBackupSize = 200;
```

Comment:
Should this be renamed to match the one in `InstrumentedProcess::InstrumentFunctions`?